### PR TITLE
Pass restaurant to view

### PR DESCRIPTION
**Pull request by JustAnOtherCoder13**
haven't find a way to get current user on restaurantDetailFragment in order to add it on click. I'll work around that.

### CODE REVIEW

File: app/src/main/java/com/picone/go4lunch/presentation/ui/restaurant/RestaurantDetailFragment.java [hunk 1]
```diff
@@ -28,16 +37,25 @@ public void onCreate(Bundle savedInstanceState) {
...
     private void initRecyclerView() {
-        RecyclerView.LayoutManager linearLayoutManager = new LinearLayoutManager(getContext());
-        mBinding.recyclerViewRestaurantDetail.setLayoutManager(linearLayoutManager);
+        mAdapter = new ColleagueRecyclerViewAdapter(mInterestedUsers);
+        mBinding.recyclerViewRestaurantDetail.setLayoutManager(new LinearLayoutManager(getContext()));
         mBinding.recyclerViewRestaurantDetail.setAdapter(mAdapter);
     }
 
-
+    static void updateRestaurantDetailUi(Restaurant restaurant) {
```
Comment (fodeguirassy) [inline app/src/main/java/com/picone/go4lunch/presentation/ui/restaurant/RestaurantDetailFragment.java]: Static for what ?

**Review by fodeguirassy — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/picone/go4lunch/presentation/ui/restaurant/RestaurantDetailFragment.java (hunk 1 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/picone/go4lunch/presentation/ui/restaurant/RestaurantDetailFragment.java]: To pass restaurant from RestaurantListfragment to RestaurantDetailFragment. Do i have to pass the object with an event or something like this?

Comment (fodeguirassy) [reply app/src/main/java/com/picone/go4lunch/presentation/ui/restaurant/RestaurantDetailFragment.java]: Use a livedata

Comment (JustAnOtherCoder13) [reply app/src/main/java/com/picone/go4lunch/presentation/ui/restaurant/RestaurantDetailFragment.java]: ok done

**Comment by JustAnOtherCoder13** [comment]
Pass almost all variable in live data. refactor methods and classes, write restaurants in firebase. change restaurant model in order to store interestedColleague list, but that list doesn't appear in database.

I must have a problem in my querry to return interested colleague list because return data is restaurant name instead of user name and photo.

**Pull request closed without merging**